### PR TITLE
Cease proxying warning. Fixes #22

### LINKED ISSUE #22: DEPRECATION: object proxying in ember 1.11.0  will
I'm using using ember beta and I'm getting the following error for routes doesn't have breadCrumb property in it's controllers :

"DEPRECATION: You attempted to access breadCrumb from (generated home.error controller), but object proxying is deprecated. Please use model.breadCrumb instead.

### PR DESCRIPTION
**Pull request by WooDzu**
Fix #22

### CODE REVIEW

File: addon/components/bread-crumbs.js [hunk 1]
```diff
@@ -34,6 +34,9 @@ export default Ember.Component.extend({
     var breadCrumbs = [];
 
     controllers.forEach(function(controller, index) {
+      if (!controller.hasOwnProperty('breadCrumb')) {
```
Comment (msalahz) [inline addon/components/bread-crumbs.js]: @WooDzu , I think this line should be :

**Comment by WooDzu** [comment]
Good spot @msalahz. I updated the pr

**Comment by msalahz** [comment]
Thank you @WooDzu

**Comment by chrisfarber** [comment]
Cool, this is a good idea. Based on some experimenting, it looks like it works correctly for computed properties/etc. There are still cases where it can display the deprecation warning (e.g., if `breadCrumb` is not defined but `breadCrumbs` is), but I can take care of that. I'll get this merged in sometime this week.

**Comment by msalahz** [comment]
Great, Thanks @chrisfarber

**Comment by pmdarrow** [comment]
Anything I can do to help move this along? @chrisfarber

**Comment by chrisfarber** [comment]
I've reverted this for 0.1.6. See Issue #27.

**Pull request merged**